CONTRIBUTING.md

```diff
@@ -119,6 +119,7 @@ bolt upgrade electron-wix-msi@latest
   - This will commit the changes automatically. Run `git log` to confirm that the changes have been
     committed.
   - `$NEW_VERSION` should be an un-prefixed [semantic version](https://semver.org/) number (e.g. `6.0.0-beta.67)
+- run `git clean -fdx`
```

Comment:
```suggestion
- Run `git clean -fdx` - this will ensure unneeded build files (and potentially sensitive files) are not included in the npm package.
```

Sorry for the nit, just wanted to clarify why this is important.